services/dynamic-sidecar/requirements/_base.in

```diff
@@ -23,3 +23,4 @@ aiofiles
 PyYAML
```

Comment:
Based in what you added in ``dev.txt, ci.txt, prod.txt`` you also need to add thisone in ``_base.in``
```
--requirement ../../packages/simcore-sdk/requirements/_base.in
```

Reply:
done this, and it also brought in new dependencies.